cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/spark/AbstractSparkContext.java

```diff
@@ -259,17 +257,15 @@ public DiscoveryServiceClient getDiscoveryServiceClient() {
    * Returns the underlying {@link Metrics} instance for emitting user metrics. The returned instance is
    * not serializable and shouldn't be exposed to user program directly.
    */
-  public Metrics getProgramWorkflowMetrics() {
-    return workflowMetricsContext == null ?
-      new ProgramUserMetrics(programMetricsContext) :
-      new WorkflowMetrics(programMetricsContext, workflowMetricsContext);
+  public Metrics getUserMetrics() {
```

Comment:
andreas had a comment about not calling it as `userMetrics` as we don't want to expose this to user program directly.